src/main/cpp/states/indexer/IndexerStateMgr.cpp

```diff
@@ -89,7 +89,11 @@ void IndexerStateMgr::CheckForStateTransition()
 
         Logger::GetLogger()->ToNtTable(m_indexer->GetNetworkTableName(), string("Ball Present"), ballPresent ? string("true") : string("false"));
 
-        if(!ballPresent)
+        if (controller->IsButtonPressed(TeleopControl::FUNCTION_IDENTIFIER::MANUAL_INDEX))

```

Comment:
should probably be if (controller != nullptr && controller->IsButtonPressed(TeleopControl::FUNCTION_IDENTIFIER::MANUAL_INDEX))